sbin/build.sh

```diff
@@ -25,9 +25,10 @@ WORKING_DIR=$1
 TARGET_DIR=$2
 OPENJDK_REPO_NAME=$3
 JVM_VARIANT=${JVM_VARIANT:-server}
-OPENJDK_UPDATE_VERSION=$5
```

Comment:
What is the reason for putting this in as the fifth parameter instead of adding it to the end? While `build.sh` isn't often called outside `makejdk.sh` this would cause a backwards compatibility issue for anyone who is invoking it directly

Reply:
I was concerned that blank variables weren't showing up on the command line in Jenkins, and that they might throw off the strict ordering requirement build.sh has. Missed the fact that the echo statement lacks the speech marks that act as placeholders in the actual command.